packages/backend/src/misc/create-temp.ts

```diff
@@ -8,3 +8,12 @@ export function createTemp(): Promise<[string, any]> {
 		});
 	});
 }
+
+export function createTempDir(): Promise<[string, any]> {
+	return new Promise<[string, any]>((res, rej) => {
```

Comment:
```suggestion
export function createTempDir(): Promise<[string, () => void]> {
	return new Promise<[string, () => void]>((res, rej) => {
```

Maybe we can avoid `any` here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/75f98a0b191aa1458658b90b5b1329748cc3c082/types/tmp/index.d.ts#L121

Reply:
I also fixed the use of `any` for the `createTemp` function, since the same applies there.